cfgov/v1/tests/wagtail_pages/test_page_creation.py

```diff
@@ -0,0 +1,259 @@
+from datetime import date
+
+from django.test import Client, TestCase
+
+from wagtail.core.models import Site
+
+from v1.tests.wagtail_pages.helpers import (
+    create_blog_page, create_browse_filterable_page, create_browse_page,
+    create_landing_page, create_learn_page, create_sublanding_filterable_page,
+    create_sublanding_page, get_parent_route
+)
+
+
+django_client = Client()
```

Comment:
One minor thing, the Django test client is available on any `django.test.TestCase` object — so, this isn't necessary, you can just reference `self.client` in the tests below here.